siliconcompiler/remote/client.py

```diff
@@ -355,6 +354,27 @@ def check_progress(chip):
         return [], True
 
 
+###################################
+def update_entry_manifests(chip):
+    '''
+    Helper method to update locally-run manifests to include remote job ID.
+    '''
+
+    flow = chip.get('option', 'flow')
+    jobid = chip.get('record', 'jobid')
+    design = chip.get('design')
+
+    entry_nodes = chip._get_flowgraph_entry_nodes(flow=flow)
```

Comment:
Yeah, I like the idea of having one `[design].pkg.json` manifest in the root build directory which we keep up to date as the job progresses. It shouldn't be too complicated to implement, we just need to be careful to avoid race conditions with parallel flowgraphs.

Reply:
Sure. Created SC-612 for it.